package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "jupyter",
     "displayName": "Jupyter",
-    "version": "2021.7.0-rc",
+    "version": "2021.6.0-rc",
```

Comment:
Change log is 6.1, do we need that here as well?

Reply:
I could, I don't think it matters

Reply:
Suppose we update it before release in any case.